bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)[\s*|✓]*(?P<severity>'
```

Comment:
I tested the unicode manually to make sure that everything works (which is anyway proved above :stuck_out_tongue: )
![regex_edit](https://user-images.githubusercontent.com/14862976/27186382-d1c9f52e-5205-11e7-8b7c-aa6e0d3513b3.png)